settings.ini

```diff
@@ -1,9 +1,9 @@
 [versions]
 FIRMWARE_VERSION=2022.03.07.0-2
```

Comment:
This should relate to the actual firmware version from gateway-rs

Reply:
there is auto bump action for it. but will update it manually for the PR.

Reply:
If this is the case, should the auto bump have not done this? Is it actually working?

Reply:
done

Reply:
> If this is the case, should the auto bump have not done this? Is it actually working?

there hasn't been a new release of gateway-rs since it was activated for hm-gatewayrs

Reply:
But it should be looking at the FIRMWARE_VERSION here to see if it's up to date?

So clearly isn't working

Reply:
Ah I think I know why.... Schedules workflows have to be in the master branch to work...